frontend/app/src/components/helper/table/TableExpandContainer.vue

```diff
@@ -1,25 +1,39 @@
 <template>
-  <td v-if="visible" class="table-expand-container" :colspan="colspan">
-    <div class="py-4">
-      <div v-if="$scopedSlots.title" class="text-h6 mb-4">
-        <slot name="title" />
+  <fragment>
+    <td :colspan="offset" />
+    <td v-if="visible" class="table-expand-container" :colspan="colspan">
```

Comment:
![image](https://user-images.githubusercontent.com/2269732/154976720-7b28231a-3f7e-4575-8a57-de36bc7dd450.png)

The changes here seem to affect every other table that uses an expanded area.
We should probably hide the <td> if the offset is 0.